src/components/ripple/ripple.tsx

```diff
@@ -112,8 +112,9 @@ export interface IRipplesProps {
   during?: number;
   display?: string;
   color?: string;
-  onClick?: (event: React.MouseEvent<HTMLDivElement>) => any;
   className?: string;
+  children?: React.ReactNode;
+  onClick?: (event: React.MouseEvent<HTMLDivElement>) => any;
```

Comment:
here, can we try void instead?

```suggestion
  onClick?: (event: React.MouseEvent<HTMLDivElement>) => void;
```

Reply:
Fixed 😄

Reply:
thank you